src/config.rs

```diff
@@ -1,29 +1,26 @@
 use std::env;
 
 /// Created to define and use the environment variables in use by Firetrap.
-pub enum EnvVar {
+pub enum Arg {
     NoDefault(&'static str),
     WithDefault(&'static str, &'static str),
 }
 
-impl EnvVar {
+impl Arg {
```

Comment:
Can be extended to later implement the hierarchy of argument passing.

Reply:
I agree that at some point we'll have to decide how we want to do configuration, and rewrite/refactor this in that fashion, but for now this is enough.